portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase

```diff
@@ -825,6 +825,32 @@ definition {
 			locator1 = "TextInput#ANY");
 	}
 
+	@description = "created (theme SVG extensions), the theme SVG extensions admin page lists the name and type"
+	@priority = 4
+	test ThemeSvgTypeCanBeCreated {
+		property custom.properties = "feature.flag.LPS-166479=true";
+
+		task ("Given a created Client Extension") {
+			ClientExtensionGeneral.goToRemoteAppsPortlet();
+		}
+
+		task ("When type is Theme svg extensions") {
+			ClientExtensionGeneral.addType(type = "Theme SVG Spritemap");
+		}
+
+		task ("And when adding requirements,and click in publish") {
+			ClientExtensionGeneral.addApp(
+				entryName = "speitemap_example.svg",
+				entryURL = "https://liferay.github.io/liferay-frontend-projects/spritemap-svg/spritemap-pencil-swapped- with-cog.svg");
+		}
+
+		task ("Then theme svg extensions admin page lists the Name and Type") {
+			FDSCustomView.assertColumnPresent(key_field = "Name");
+
+			FDSCustomView.assertColumnPresent(key_field = "Type");
```

Comment:
Hey the confirmation is on the client extension name (speitemap_example.svg) and the type (Theme SVG Spritemap) and not on the columns.

Reply:
ok, thanks